selenium_test.py

```diff
@@ -0,0 +1,222 @@
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import Select
+import pandas as pd
+import time
+
+KEYWORDS_FILE = 'textfile.txt'
+RANKED_OUTPUT_FILE = 'Output.csv'
+START_YEAR = "1917"
+END_YEAR = "1971"
+
+
+class resultLists:
+    title_list = []
+    description_list = []
+    category_list = []
+    keyword_list = []
+    link_list = []
+
+lists = resultLists()
+
+
+
+def read_keywords():
+    with open(KEYWORDS_FILE) as fd:
+        return [keyword.strip() for keyword in fd.readlines()]
+
+
+def economist_scraper(keyword, startyear, endyear):
+    driver = webdriver.Firefox(executable_path='/Users/dylanedwards/PycharmProjects/economist-metasearcher/geckodriver')
+    driver.get("http://www.economist.com/historicalarchive")
+
+    # tests to make sure we are on login page
+    assert "Economist" in driver.title
+
+    # inputs username
+    username = driver.find_element_by_name("email")
+    username.clear()
+    username.send_keys("dylanedwards290@gmail.com")
+
+    # inputs passwords
+    password = driver.find_element_by_name("password")
+    password.clear()
+    password.send_keys("duDnis-0givza-carfed")
```

Comment:
Consider reading login information from a file (that does *not* go into the git repo).